edge/pkg/metamanager/msg_processor.go

```diff
@@ -115,7 +115,7 @@ func parseResource(resource string) (string, string, string) {
 
 // is resource type require remote query
 func requireRemoteQuery(resType string) bool {
-	return resType == model.ResourceTypeConfigmap || resType == model.ResourceTypeSecret || resType == constants.ResourceTypeService || resType == constants.ResourceTypeEndpoints
+	return resType == model.ResourceTypeConfigmap || resType == model.ResourceTypeSecret || resType == constants.ResourceTypeEndpoints
```

Comment:
We support process remoteQuery for Service. Why are we removing it ?

Reply:
There are several reasons for the changes here. First，we have to get the service endpoint every time. if we get it from the cloud every time, it is very time consuming and extremely inefficient.  Second, the update of service and endpoint is actually notified by the cloud, we did not take the initiative to watch. Every time we go to the cloud to query, and the related resource information has not changed, the operation of the active query is actually redundant. Therefore, the operation of supporting the service to query on the cloud is removed

Reply:
OK